app/main.py

```diff
@@ -17,3 +17,8 @@
 collection_of_coins = {1, 2, 25}
 
 # write your code here
+
+sorted_variables = {
+    "mutable": [my_favourite_films, marks, collection_of_coins],
+    "immutable": [lucky_number, pi, one_is_a_prime_number, name, profile_info]
+}
```

Comment:
Remove a pytest cache files from this Pull request. Always add only files that will be useful for a mentor to review your task.